.github/ISSUE_TEMPLATE/contribute.txt

```diff
@@ -0,0 +1,24 @@
+---
+name: Contribute
+about: Let us know you have an example you'd like to contribute
+
+---
+
+**Description**
+A clear and concise description of the concepts in your project and the problem your code solves. Please include any references you used. 
+
+**How to run the example**
+A short description of how to run your code, including an example. 
+
+**Link to your repo**
+Include a link to the repo you want to contribute. We will fork it. If we want to make any changes to the code we will create a pull request for you to approve.
```

Comment:
Perhaps be more explicit that the repo they are contributing is going into `dwave-examples`?
"Include a link... to contribute to `dwave-examples`"